provider/terraform/property_override.rb

```diff
@@ -26,6 +26,12 @@ module OverrideFields
       attr_reader :sensitive # Adds `Sensitive: true` to the schema
       attr_reader :validation # Adds a ValidateFunc to the schema
 
+      attr_reader :is_set # Uses a Set instead of an Array
```

Comment:
I'm fine with this for now since sets are rarer than lists, but it probably makes sense in the future to have sets be the default, and lists needing to be specified explicitly

Reply:
I agree that it might make sense. We will need to be careful with the set func for nested objects... we can revisit later and decide if we want to switch.